# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/white-space/trailing-space-position-001.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/css-text/white-space/trailing-space-position-001.ini
--- a/testing/web-platform/meta/css/css-text/white-space/trailing-space-position-001.ini	Fri Apr 02 22:03:46 2021 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,20 +0,0 @@
-[trailing-space-position-001.html]
-  [CSS Test: Positions of trailing collapsible spaces]
-    expected:
-      if os == "mac" and debug: FAIL
-
-  [CSS Test: Positions of trailing collapsible spaces 1]
-    expected:
-      if os == "mac" and debug: FAIL
-
-  [CSS Test: Positions of trailing collapsible spaces 2]
-    expected:
-      if os == "mac" and debug: FAIL
-
-  [CSS Test: Positions of trailing collapsible spaces 3]
-    expected:
-      if os == "mac" and debug: FAIL
-
-  [CSS Test: Positions of trailing collapsible spaces 4]
-    expected:
-      if os == "mac" and debug: FAIL